crates/holochain_zome_types/src/query.rs

```diff
@@ -579,4 +602,34 @@ mod tests {
             [true, false, false, false, true, true, false].to_vec()
         );
     }
+
+    #[test]
+    fn filter_by_multiple_action_types() {
+        let actions = fixtures();
+
+        assert_eq!(
+            map_query(
+                &ChainQueryFilter::new()
+                    .action_type(actions[0].action_type())
+                    .action_type(actions[1].action_type()),
+                &actions
+            ),
+            [true, true, false, true, true, true, false].to_vec()
+        );
+    }
+
+    #[test]
+    fn filter_by_multiple_entry_types() {
+        let actions = fixtures();
+
+        assert_eq!(
+            map_query(
+                &ChainQueryFilter::new()
+                    .entry_type(actions[0].entry_type().unwrap().clone())
+                    .entry_type(actions[1].entry_type().unwrap().clone()),
+                &actions
+            ),
+            [true, true, false, true, true, true, false].to_vec()
+        );
```

Comment:
Maybe extend this test with
```rust
assert_eq!(
            map_query(
                &ChainQueryFilter::new().entry_type(actions[0].entry_type().unwrap().clone()),
                &actions
            ),
            [true, false, false, false, true, true, false].to_vec()
        );
```

to test that it's actually filtering some entry types out.